tests/dags/common/loader/test_reporting.py

```diff
@@ -95,15 +95,45 @@ def _make_report_completion_contents_data(media_type: str):
 @pytest.mark.parametrize(
     "image_data, image_expected", _make_report_completion_contents_data("image")
 )
+@pytest.mark.parametrize(
+    "dated, date_range_start, date_range_end, expected_date_range",
+    [
+        # Not dated, no date range
+        (False, "2022-01-01", "2022-05-01", "all"),
+        # Not dated, but date range supplied
+        (False, None, None, "all"),
```

Comment:
Not a big deal, but I think the two comments above are switched around.

Reply:
Whoops! How'd that happen 😂 thanks!